resources/lib/pigskin.py

```diff
@@ -358,17 +357,31 @@ def get_shows_episodes(self, show_name, season=None):
         """Return a list of episodes for a show. Return empty list if none are
         found or if an error occurs."""
         url = self.config['modules']['API']['NETWORK_PROGRAMS']
-        programs_dict = self.make_request(url, 'get')
-        for show in programs_dict['modules']['programs']:
-            name = show['title']
-            if show_name == name:
-                slug = show['slug']
-                for seasons in show['seasons']:
-                    season_name = seasons['value']
-                    if season == season_name:
-                        season_id = seasons['slug']
+        programs = self.make_request(url, 'get')['modules']['programs']
+        for show in programs:
```

Comment:
I'm not sure how Python behaves here. Will this also attempt to iterate over any errors in that make_request() might return?